src/SFA.DAS.EmployerFinance.Api/Global.asax.cs

```diff
@@ -1,66 +1,21 @@
-﻿using System.Data.Common;
-using System.Web;
+﻿using System.Web;
 using System.Web.Http;
-using Microsoft.Azure;
-using Microsoft.ApplicationInsights.Extensibility;
-using NServiceBus;
-using SFA.DAS.EmployerFinance.Configuration;
-using SFA.DAS.EmployerFinance.Extensions;
-using SFA.DAS.Extensions;
-using SFA.DAS.NServiceBus;
-using SFA.DAS.NServiceBus.NewtonsoftJsonSerializer;
-using SFA.DAS.NServiceBus.NLog;
-using SFA.DAS.NServiceBus.SqlServer;
-using SFA.DAS.NServiceBus.StructureMap;
-using SFA.DAS.UnitOfWork.NServiceBus;
-using StructureMap;
-using WebApi.StructureMap;
+using SFA.DAS.EmployerFinance.Startup;
 
 namespace SFA.DAS.EmployerFinance.Api
 {
     public class WebApiApplication : HttpApplication
     {
-        private IEndpointInstance _endpoint;
-
         protected void Application_Start()
         {
             GlobalConfiguration.Configure(WebApiConfig.Register);
 
-            StartServiceBusEndpoint();
+            ((IStartup)GlobalConfiguration.Configuration.DependencyResolver.GetService(typeof(IStartup))).StartAsync().GetAwaiter().GetResult();
```

Comment:
There's a generic extension method `WebApi.StructureMap.Extensions.GetService<T>` that you can use instead that allows you to avoid the cast if you prefer.